test/unit/test_test.gd

```diff
@@ -1447,4 +1447,234 @@ class TestMemoryMgmt:
 		var TestClass = load('res://addons/gut/test.gd')
 		for i in range(3):
 			var extra_test = TestClass.new()
-			add_child(extra_test)
\ No newline at end of file
+			add_child(extra_test)
+
+
+class TestAssertSetgetCalled:
+	extends BaseTestClass
+	
+	
+	const NodeChildMock = preload("res://test/resources/test_assert_setget_test_objects/node_child_mock.gd")
+	const SceneMock = preload("res://test/resources/test_assert_setget_test_objects/SceneMock.tscn")
+	const ControlChildMock = preload("res://test/resources/test_assert_setget_test_objects/ControlChildMock.tscn")
+	const InheritSceneMock = preload("res://test/resources/test_assert_setget_test_objects/InheritSceneMock.tscn")
+	
+	func before_each():
+		.before_each()
+		#orphans hunting => ask butch how to handle this
+		gr.test_with_gut.gut.add_child(gr.test_with_gut.gut._yield_timer)
+		gr.test_with_gut.gut.add_child(gr.test_with_gut.gut._yield_between.timer)
+		gr.test_with_gut.gut.add_child(gr.test_with_gut.gut._wait_timer)
```

Comment:
GREAT find.  I did a quick check and this kills almost all (704 -> 35) the orphans in this script.  I'll move this into `.before_each` after I merge.

Reply:
Ok, cool. Also I created a related ticket for this issue now.

In the long run this should be something the gut class takes care of, I think. Since it can affect code outside of `test_test.gd`, I think. Also the attributes are labeled for internal use only, if I understand correctly.

Reply:
They are for  internal use.  GUT adds the timers in `_ready` but for these tests GUT is never getting added as a child  so they end up being orphans.  The "right" fix might be to  add GUT as a child.